client/src/components/exercise/lab9/pages/Repair.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-unused-vars */
```

Comment:
If this is just a test file maybe we don't need it in the PR

Reply:
Commented out currently unused variables in the Repair that still need to be implemented